ksql-common/src/main/java/io/confluent/ksql/configdef/ConfigValidators.java

```diff
@@ -33,11 +33,14 @@ private ConfigValidators() {
   }
 
   public static <T extends Enum<T>> Validator enumValues(final Class<T> enumClass) {
-    final String[] validValues = EnumSet.allOf(enumClass)
+    final String[] enumValues = EnumSet.allOf(enumClass)
```

Comment:
`VALUE_FORMAT` for C*AS statements are optional, so `null` must be a valid value for Format.  

More generically, validators should allow `null` as a value because the definition will determine if `null` is allowed or not by looking at the `defaultValue`.  If it's `ConfigDef.NO_DEFAULT_VALUE` that `null` is not allowed, otherwise it is.  Hence `null` checking the responsibility of `ConfigDef` via `defaultValue` and NOT the responsibility of the validators.